docs/credentials_and_secrets.md

```diff
@@ -0,0 +1,72 @@
+# Credentials and Local Secrets
+
+Rails credentials allows you to securely store sensitive configuration information and credentials for your Rails application. This feature is particularly useful for storing things like API keys, database passwords, or any other confidential data that your application needs to access.
+
+Local secrets (environment variables) are secrets that are environment-specific information unique to your local system or the cloud platform you use such as Heroku. We use these variables when every local system requires different credentials. They can also be used to quickly change certain settings that might change very often like disabling a certain feature temporarily. There is no need to update any code and re-deploy the app in this case.
```

Comment:
link to the official documentation on how credentials file works. 

in here we just need to say how we use that credentials file and also .env file for local variables. 

talk about the difference for when we would use either. 
* credential files - when the same keys needs to be shared by all devs and want keys to be in version control
* .env file variables that can be access with ENV[''] method - env specific - both for each developer in their machines or if they have multiple deployed servers, server specific keys